widgets/get_cp2k_input.py

```diff
@@ -752,6 +753,15 @@ def get_force_eval_qs_dft(self):
                 },
                 'SCF': scf_opt[self.inp_dict['diag_method']],
                 'XC': self.sections_dict[self.workchain]['xc'],
+                # Always generate density/spin cube
```

Comment:
when will this write the cube file? If it is only at the end of the geometry optimization run then it's OK
if it is at each geo opt step it's not nice

Reply:
Described here https://manual.cp2k.org/cp2k-8_1-branch/CP2K_INPUT/FORCE_EVAL/DFT/PRINT/E_DENSITY_CUBE/EACH.html